autoload/fzf/vim.vim

```diff
@@ -41,7 +41,7 @@ if s:is_win
   else
     let s:bin.preview = fnamemodify(s:bin.preview, ':8')
   endif
-  let s:bin.preview = escape(s:bin.preview, '\')
+  let s:bin.preview = 'bash '.escape(s:bin.preview, '\')
```

Comment:
What happens if we prepend `'ruby '` here when `executable('ruby')` is true? Can you elaborate more on why Ruby script cannot be used on Windows?

Reply:
Highlighting the current row didn't work when I ran the ruby script inside a batchfile.

My original plan is to make a batchfile, `preview.cmd`, to resolve the network drive via `cd /d` and then call the ruby script via `ruby .\preview.rb` or bash script via `bash .\preview.sh`. cmd.exe does not know about shebang and relies on associations between an extension and an executable to execute anything besides `*.exe`. I made one to resolve the network drive for both preview scripts without modifying them and they both run but highlighting didn't work for the ruby script.

What's strange is that highlight worked if the user associated the `rb` extension with ruby and then ran the preview script directly. It works for `fzf#vim#ag`.

Reply:
Okay, let me merge this for now. Please let me know if you happen to find a solution.